metatron-app/metatron-discovery
# #768 autocompletion of editing rule for window functions

### Description

5 window functions added.
autocompletion supports to the function names

**Related Issue** : 

[#768](https://github.com/metatron-app/metatron-discovery/issues/768)

### How Has This Been Tested?

1. enter rule editing screen
2. add a rule for a window command
3. focus the condition input text field
4. window function names should be proposed  
5. add a rule for a aggregate command
6. focus the condition input text field
7. there is no window function name

All processes should be normal without abnormalities.
 

### Types of changes



### Checklist:



### Additional Context
